src/components/DatasetSelector.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+
+function DatasetSelector(props) {
+    
+    /* For testing - remove later. These will either be passed in as props or come from the store. */
+    const options = [
+        'Poverty Rates',
+        'Food Insecurity',
+        'WIC Usage',
+        'Snap Usage',
+        'Census',
+    ];
+
+	// const { options } = props;
+
+    const [selectedOption, setSelectedOptions] = useState(
+		options[0] ? options[0] : null,
+	);
+
+	const handleSelection = (e, index) => {
+		console.log(e);
+		console.log(index);
```

Comment:
Unless there is a reason to leave them in, can you take our your console logs? Also the comment in line 14.